### ISSUE TITLE
Camera stream orientation not always honored on picture-elements card

### ISSUE THREAD

**Issue by TarheelGrad1998** [home-assistant/frontend#15539]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Adding a picture elements card for a camera entity where the stream orientation has changed does not apply the orientation unless camera_view is live.  

### Describe the behavior you expected

Camera stream orientation is always honored

### Steps to reproduce the issue

1. Set Camera stream orientation for a camera
2. Add picture elements card without camera_view parameter
3. View camera orientation.

### What version of Home Assistant Core has the issue?

2023.2.5

### In which browser are you experiencing the issue with?

Google Chrome

### Which operating system are you using to run this browser?

Windows 10

### Problem-relevant frontend configuration

```yaml
camera_image: camera.kitchen_doorbell
tap_action: more-info
type: picture-elements
```

**Comment by TarheelGrad1998** [comment]
Not resolved.

**Comment by TarheelGrad1998** [comment]
Nope, it still broken.  Be nice to see it fixed eventually instead of just allowed to auto close.

**Comment by TarheelGrad1998** [comment]
Still not resolved

**Comment by TarheelGrad1998** [comment]
Still not resolved in 2024.4

**Comment by TarheelGrad1998** [comment]
I've been checking on this every time the bot triggers, but I'm not going to bother this time.  I'm certain this isn't fixed, and I'm also certain at this point there is zero chance that it ever will be, after being ignored for a year and a half.  It's a minor issue, sure, but it would be great for it to be fixed all the same.  I give up.
